nautobot/project-static/js/rack_elevations.js

```diff
@@ -11,3 +11,82 @@ $('button.toggle-images').click(function() {
     $(this).children('span').toggleClass('mdi-checkbox-marked-circle-outline mdi-checkbox-blank-circle-outline');
     return false;
 });
+
+
```

Comment:
Conversations around future UI have discussed ditching jQuery in core. Vanilla JavaScript used to implement this functionality.